src/cls/IPM/Repo/UniversalSettings.cls

```diff
@@ -106,8 +106,11 @@ ClassMethod GetDefaultValue(indx As %String = "") As %String
     Return $Get(^IPM.settings(indx,"default"))
 }
 
-ClassMethod SetValue(indx As %String = "", val As %String = "") As %Status
+ClassMethod SetValue(indx As %String = "", val As %String = "", overwrite As %Boolean = 1) As %Status
 {
+    if ('$GET(overwrite, 1)) && ..GetValue(indx)'="" {
```

Comment:
In general, my rule is to always put parentheses around things with binary operators. So this would be:
`if '$Get(overwrite,1) && (..GetValue(indx) '= "")`

Reply:
Could also be worth adding a unit test (not integration test) around the overwrite/not behavior.